tools/ghc.bzl

```diff
@@ -0,0 +1,26 @@
+load("@bazel_tools//tools/cpp:lib_cc_configure.bzl", "get_cpu_value")
+
+default_ghc_workspaces = {
+  "k8": "@ghc",
+  "darwin": "@ghc",
+  "x64_windows": "@ghc_windows",
+}
+
+def get_ghc_workspace(ghc_workspaces, repository_ctx):
+  """Return the GHC workspace appropriate for the current OS."""
+  cpu_value = get_cpu_value(repository_ctx)
```

Comment:
Is this preferable to `repository_os`? https://docs.bazel.build/versions/master/skylark/lib/repository_os.html#name

Reply:
Ah, after reading the source for get_cpu_value I see why you've done it this way, os.name is pretty messy.